src/Client/Updater.php

```diff
@@ -31,40 +31,38 @@ class Updater
   /**
    * Updater constructor.
    *
-   * @param string $repository_name
+   * @param string $repositoryName
    * @param array[][] $mirrors
    *   Re
    */
-    public function __construct($repository_name, $mirrors)
+    public function __construct($repositoryName, $mirrors)
     {
-        $this->repoName = $repository_name;
+        $this->repoName = $repositoryName;
         $this->mirrors = $mirrors;
     }
 
 
   /**
    * @todo Update from python comment https://github.com/theupdateframework/tuf/blob/1cf085a360aaad739e1cc62fa19a2ece270bb693/tuf/client/updater.py#L999
-   *
-   * @param bool $unsafely_update_root_if_necessary
    */
-    public function refresh($unsafely_update_root_if_necessary = true)
```

Comment:
Technically out of scope. Can we file a separate issue to refer back to the Python implementation and evaluate whether we need this, and put a `@todo` comment in the code referencing the issue?